app/services/submit_to_app_store/prior_authority_payload_builder.rb

```diff
@@ -5,20 +5,15 @@ def initialize(application:)
     end
 
     def payload
-      content = data
```

Comment:
By deleting this you call `data` twice, which means that the payload has to be fully constructed from scratch twice. If you don't like stashing it in a variable here, can I suggest you memoise the `data` method by starting it off with `@data ||=`?